tests/e2e/config.json

```diff
@@ -1,7 +1,7 @@
 {
-    "sfUrl": "",
+    "sfUrl": "https://devmagazine-sit.sitefinity.site/",
```

Comment:
exclude this file

Reply:
yes, i just noticed it, it should be ok now